webapp/style/_vars.scss

```diff
@@ -33,6 +33,7 @@ $zIndex20Dropdown: 20;
 $zIndex80AppNotification: 80;
 $zIndex90AppError: 90;
 $zIndex190AppLoader: 190;
+$zIndex900UserMenu: 900;
```

Comment:
everything should go below appLoader, so maybe this can be ```$zIndex180UserMenu: 180;```

Reply:
My use case for having this be the topmost element was to be able to do something in case the loader takes too long / forever.

But the policy then is to disallow operations while something is loading?

Reply:
yes,  we show the appLoader to stop any operation from the user...